core/src/stores/data_cache.rs

```diff
@@ -43,6 +47,38 @@ impl DataCache {
 
         self.tx_subs.clean(ttl_duration);
     }
+
+    pub async fn check_if_confirmed_or_expired_blockheight(
+        &self,
+        sent_transaction_info: &SentTransactionInfo,
+    ) -> bool {
+        self.txs
+            .is_transaction_confirmed(&sent_transaction_info.signature)
+            || self
+                .block_store
+                .get_latest_block(CommitmentConfig::processed())
+                .await
+                .block_height
+                > sent_transaction_info.last_valid_block_height
+    }
+
+    pub fn new_for_tests() -> Self {
```

Comment:
could add cfg test 🤔

Reply:
means?

Reply:
Cannot use cfg test, they are not in same crate.

https://stackoverflow.com/questions/72901643/cant-call-function-in-unit-test-from-integration-test-code